### PR TITLE
[Feat] docs: Create howto-use-mermaid-graphic-tool.md

### PR DESCRIPTION
**Pull request by pitdagosti**
Creation of an HOWTO to document how to use the Mermaid graphic tool.

Fix https://github.com/B-AROL-O/ARNEIS/issues/179

### CODE REVIEW

**Review by gmacario — COMMENTED** [review]
Let me rework the title to make sure it will be consistent with the other HOWTOs

image

**Review by gmacario — APPROVED** [review]

**Comment by gmacario** [comment]
LGTM, asking @alv67 to have a look as well.

File: docs/howto/howto-use-mermaid-graphic-tool.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# HOWTO Mermaid utilization and code examples
+
+[Mermaid](https://mermaid-js.github.io/mermaid/#/) is a useful tool to create diagrams and charts directly using [Markdown](https://daringfireball.net/projects/markdown/).
+The good part about it is that those diagrams can be easily modified just by changing a code displayed before the image, however we can't just move the image wherever we want to.
+For this reason the [Mermaid Live Editor](<https://mermaid.live/>) is better in case we are just interested in the results.
+
+In order to create a Mermaid graph:
+
+* Use the bash code (```bash```) and instead of writing "bash" write "mermaid".
```
Comment (alv67) [inline docs/howto/howto-use-mermaid-graphic-tool.md:9]: Change in something like:
* Use a _code block_ tagged with language `mermaid`

**Review by alv67 — CHANGES_REQUESTED** [review]
See comment on code

**Review by alv67 — APPROVED** [review]

**Pull request merged**